src/index.html

```diff
@@ -20,8 +20,402 @@
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="image"
+            src="images/imac.jpeg"
+            alt="imac"
+          />
+          <div class="content-wrapper">
+            <div class="product-name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </div>
+            <div class="product-code">Product code: 195434</div>
+            <div class="content-revie">
+              <div class="stars">
+                <div class="stars stars--4">
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                </div>
+              </div>
+              <div class="revie-text">Reviews: 5</div>
+            </div>
+            <div class="content-price">
+              <div class="price-text">Price:</div>
+              <div class="prise-num">$2,199</div>
+            </div>
+            <a
+              class="button-link"
+              href="buy"
+              data-qa="card-hover"
+            >
+              <button class="button">BUY</button>
+            </a>
+          </div>
+        </div>
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="image"
+            src="images/imac.jpeg"
+            alt="imac"
+          />
+          <div class="content-wrapper">
+            <div class="product-name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </div>
+            <div class="product-code">Product code: 195434</div>
+            <div class="content-revie">
+              <div class="stars">
+                <div class="stars stars--4">
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                </div>
+              </div>
+              <div class="revie-text">Reviews: 5</div>
+            </div>
+            <div class="content-price">
+              <div class="price-text">Price:</div>
+              <div class="prise-num">$2,199</div>
+            </div>
+            <a
+              class="button-link"
+              href="buy"
+              data-qa="hover"
```

Comment:
Inconsistent use of data attributes. The attribute 'data-qa' is used as 'data-qa="hover"' here, while it is 'data-qa="card-hover"' in other places. Ensure consistent naming for data attributes.